log/martianlog/martianlog.go

```diff
@@ -7,10 +7,34 @@
 package martianlog
 
 import (
+	"context"
+
+	"github.com/saucelabs/forwarder/internal/martian"
 	martianlog "github.com/saucelabs/forwarder/internal/martian/log"
 	"github.com/saucelabs/forwarder/log"
 )
 
 func SetLogger(l log.Logger) {
-	martianlog.SetLogger(l)
+	martianlog.SetLogger(martian.TraceIDPrependingLogger{
+		Logger: contextLogger{l},
+	})
+}
+
+// contextLogger is a wrapper around log.Logger that implements the martian log.Logger interface.
+type contextLogger struct {
+	log.Logger
+}
+
+var _ martianlog.Logger = contextLogger{}
+
+func (l contextLogger) Errorf(_ context.Context, format string, args ...interface{}) {
```

Comment:
Let's use `any` instead of `interface{}`. 

BTW the linter should have caught it.